WordPress/src/main/java/org/wordpress/android/ui/posts/editor/media/EditorMedia.kt

```diff
@@ -295,7 +295,7 @@ class EditorMedia @Inject constructor(
                 }
         }
         analyticsTrackerWrapper.track(EDITOR_UPLOAD_MEDIA_FAILED, properties)
-        listener.onMediaUploadFailed(media.id.toString())
+        listener.onMediaUploadFailed(media.id.toString(), error.type.name)
```

Comment:
Passing solely the `error.type.name` string feels a bit messy, but when I attempt to pass `error` and I am unable to leverage the relevant types from the FluxC package, presumably because the editor files are in a separate `lib` from the main app. 

Is it possible to access the `MediaError` type from the FluxC package?

Reply:
> Is it possible to access the MediaError type from the FluxC package?

Yes. In other areas, `MediaErrorType` is imported directly from the `MediaStore`. However, we already have a precedent of sending `error.type.name` from `MediaError` to the analytics in this function, so I'm not certain if there's an advantage of importing `MediaErrorType` directly here. It seems fine to pass `error.type.name` to `onMediaUploadFailed` here, but certainly open to using `MediaErrorType` if you see an advantage.

https://github.com/wordpress-mobile/WordPress-Android/blob/234db297728a506ee8e9c318d42c9cd9e7e941b3/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L107-L108

Reply:
As noted in https://github.com/wordpress-mobile/WordPress-Android/pull/19878#discussion_r1441396013, we'd avoid the need to pass `error.type.name` entirely in https://github.com/wordpress-mobile/WordPress-Android/pull/19884.